test/functional/shell/viml_system_spec.lua

```diff
@@ -195,15 +231,10 @@ describe('system()', function()
       end)
     end
   end)
-
-  describe('command passed as a list', function()
-    it('does not execute &shell', function()
-      eq('* $NOTHING ~/file',
-         eval("system(['echo', '-n', '*', '$NOTHING', '~/file'])"))
-    end)
-  end)
```

Comment:
Ha, I forgot we supported this (doesn't seem to be documented). It's useful, like a synchronous `jobstart()`. Though I guess it violates the analogy of `system()` from C stdlib.